pkgs/tools/system/disk-filltest/default.nix

```
@@ -0,0 +1,38 @@
+{ stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  pname = "disk-filltest";
+  version = "0.8.1";
+
+  src = fetchFromGitHub {
+    owner = "bingmann";
+    repo = pname;
```

```suggestion
    repo = "disk-filltest";
```
`repo` and `pname` are the same, but they don't necessarily change together.